app/services/remediations_api.rb

```diff
@@ -0,0 +1,67 @@
+# frozen_string_literal: true
+
+# This class is meant to contain all calls to the Remediations API.
+class RemediationsAPI
+  def initialize(account, url)
+    @url = URI.parse(url)
+    @b64_identity = Base64.strict_encode64(account.to_identity_header.to_json)
+  end
+
+  def import_remediations
+    Rule.find_in_batches(batch_size: 100).each do |rules|
+      response = Platform.connection.post(@url) do |req|
+        req.headers['X_RH_IDENTITY'] = @b64_identity
```

Comment:
This stuff should come from the connection automatically, right? I think you can override things specified in the connection here that you'd like to change just for this request. Things like 'Content-Type' header could probably be moved to the connection.

Reply:
It would be nice to have some defaults but the list of default params for the connection are very limited - https://lostisland.github.io/faraday/middleware/list - everything else is on a per-request basis.